client/packages/programs/src/JsonForms/common/components/Array.tsx

```diff
@@ -254,7 +297,113 @@ const ArrayComponent = (props: ArrayControlCustomProps) => {
     label,
     rootSchema,
     renderers,
+    config,
   } = props;
+  const { errors: zErrors, options } = useZodOptionsValidation(
+    Options,
+    props.uischema.options
+  );
+
+  const defaultExpanded = props.uischema?.options?.['defaultExpanded'] ?? false;
+
+  const [expandedItems, setExpandedItems] = useState<boolean[]>(
```

Comment:
An array of which elements are expanded or aren't (booleans, accessed by index).

These needed to become controlled components so I could know what the current state of each accordion was (since the Notes label is dependent on it)